tests/config/app_topic_subscription_pluggable_pubsub.yaml

```diff
@@ -11,13 +11,14 @@
 # limitations under the License.
 #
 
-apiVersion: dapr.io/v1alpha1
```

Comment:
It’s important to test with at least one on version v1alpha1 so we test the upgrading. If you change them all, we are removing testing some functionality.

the reason it fails for you is generally due to not having the CRDs installed or not having dapr running correctly on the cluster.

Reply:
Thanks @ItalyPaleAle , I realized you're right that is because I didn't have dapr runtime running before the e2e.